.github/workflows/one-step.yml

```
@@ -4,9 +4,8 @@ name: One-Step
 
 # Controls when the action will run.
 on:
-  # Triggers the workflow on push or pull request events but only for the master branch
-  push:
-    branches: [feat/ibc-wasm]
+  release:
+    types: [fix/hmstr]
```

actionlint

invalid activity type "fix/hmstr" for "release" Webhook event. available types are "created", "deleted", "edited", "prereleased", "published", "released", "unpublished"

(events)